ipapython/config.py

```diff
@@ -71,7 +70,17 @@ def check_ip_option(option, opt, value):
     try:
         return CheckedIPAddress(value)
     except Exception as e:
-        raise OptionValueError("option %s: invalid IP address %s: %s" % (opt, value, e))
+        raise OptionValueError("option %s: invalid IP address %s: %s" %
+                               (opt, value, e))
+
+def check_ip_option_with_loopback(option, opt, value):
```

Comment:
It's more straightforward to code, not to maintain. Once someone comes saying he does not like the error message, you go and change it on one spot, forget about the other, this goes to Bugzilla, QA will push it back to you, and you have to fix it once more.
Having `functools.partial` here would allow you to prevent this circle.
But if you don't want it there, fine, I won't hold this PR just because of that.